### PR TITLE
Update dashboards

### LINKED ISSUE #5818: Update dashboards
# Description

Following the release of our integrations' documentation, we want to incorporate the new dashboards into our main application.

## Functional requirements

- Users must be able to visualize and interact with the new dashboards
- Users must be able to interact with the search bar and get dashboard contents updated
- Users must be able to interact with the time selector of a visualization and all visualization of the dashboard must update with the new selection
- Users without the appropriate roles should not be able to view a dashboard

## Non-functional requirements

- All dashboards must update to changes of the search bar
- All dashboards must have matching colors for the same information
- All dashboards must have limited the amount of information they can display, so there is no overflow of legends or lines
- All titles and texts must use the i18n framework if possible
- All queries must take into account the size of the information they are going to display, to limit the max amount of information when it makes sense, in order to preserve the responsiveness of the dashboard
- All queries must be documented in the dashboard implementation guide

## Implementation restrictions

- Update the dashboards' implementation guide
- Use built-in styles and components 
- We must clean the current design, introducing the appropriate guards and tests to ensure the correct rendering even if the available data is not correct

## Plan

- [x] Create dashboards definitions
  - [x] wazuh/wazuh-kibana-app#5838
  - [x] wazuh/wazuh-kibana-app#5898
  - [x] wazuh/wazuh-kibana-app#5909
  - [x] wazuh/wazuh-kibana-app#5911
  - [x] https://github.com/wazuh/wazuh-kibana-app/issues/5763
    https://documentation.wazuh.com/4.3/_images/screenshot-05-Wazuh-Vulnerability-Detection1.png
  - [x] wazuh/wazuh-kibana-app#5930
  - [x] wazuh/wazuh-kibana-app#5933
  - [x] wazuh/wazuh-kibana-app#5938
  - [x] wazuh/wazuh-kibana-app#5940
  - [x] wazuh/wazuh-kibana-app#5941
 - [x] Update dashboards implementation with the new definitions
 - [x] wazuh/wazuh-dashboard-plugins#6066
 - [x] wazuh/wazuh-kibana-app#6071
 - [x] wazuh/wazuh-kibana-app#6074
 - [x] wazuh/wazuh-kibana-app#6079
 - [ ] ~~After implementing the dashboard log data analysis in our plugin, we will have to develop a flyout when clicking on the data in the table.~~
 - [ ] ~~After implementing the dashboard integrity monitoring in our plugin, we will have to develop a flyout when clicking on the data in the table.~~
 - [x] Add the necessary changes to make easier the dashboard customization
- [x] Add all the related visualizations to the dashboard (to see and decide which visualizations will be added to the definitive dashboard)
- [x] Create a update dashboard guide to use in the next dashboards updates

### PR DESCRIPTION
**Pull request by chantal-kelm**
### Description

Update all dashboards definitions
 
### Issues Resolved

#5818 

### Evidence
[Provide screenshots or videos to prove this PR solves the issues]

### Test

Verify that all dashboards are up to date.

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [x] New functionality includes testing.
- [x] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: plugins/main/server/integration-files/visualizations/overview/overview-fim.ts
Comment (JuanGarriuz) [inline plugins/main/server/integration-files/visualizations/overview/overview-fim.ts:1]: Is commented code necessary, or can it be removed?

**Review by JuanGarriuz — APPROVED** [review]
LGTM!!

image

image

**Review by Machi3mfl — APPROVED** [review]
## Security Events

Screenshot 2023-11-02 at 16 53 43

Screenshot 2023-11-02 at 16 57 01


## Policy Monitoring

Screenshot 2023-11-02 at 16 54 38

Screenshot 2023-11-02 at 16 57 37

## SCA

Screenshot 2023-11-02 at 16 55 23

Screenshot 2023-11-02 at 16 58 05


## Docker

Screenshot 2023-11-02 at 16 56 16

Screenshot 2023-11-02 at 16 58 34

**Review by asteriscos — APPROVED** [review]
CR: :heavy_check_mark: 
Test: :heavy_check_mark: 

image

image

**Pull request merged**